daffodil-tdml-processor/src/main/scala/org/apache/daffodil/tdml/processor/DaffodilTDMLDFDLProcessor.scala

```diff
@@ -126,16 +125,21 @@ final class TDMLDFDLProcessorFactory private (
       val diags = p.getDiagnostics
       Left(diags)
     } else {
-      val dp =
+      val dp = {
+        // Always serialize the parser/unparser objects.
+        // This insures nothing is left uncomputed by lazy evaluation.
```

Comment:
Mentioned this earlier, but this sounds like a bug to me. We shouldn't need to serialize the parser to ensure everything is  evaluated.

Reply:
I've removed this from here. It now only saves and reloads if useSerializedProcessor. 
This is the compiler's problem to insure that everything is fully evaluated, not code out here using the data processor.